README.md

```diff
@@ -412,3 +411,26 @@ No additional parameters required
 ### Get Tags
 
 `GET /api/tags`
+
+## Getting started
+
+### 1. Installation steps
+
+- Clone the repo using
+
+`https://github.com/andela/savage_rangers-ah.git`
+
+- Install dependecies by runing
```

Comment:
@nkpremices small typo here: `dependencies`

Reply:
Oh!!! Thank you @MemunaHaruna